package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "4.7.17",
+  "version": "4.8.0",
```

Comment:
There is a lot of cleanup in here so it feels more like a minor release than a patch. Do you agree?

Reply:
Agree!